src/connector/mysql/conversion.rs

```diff
@@ -105,17 +106,33 @@ impl TypeIdentifier for my::Column {
         matches!(self.column_type(), MYSQL_TYPE_DOUBLE)
     }
 
-    fn is_integer(&self) -> bool {
+    fn is_int32(&self) -> bool {
         use ColumnType::*;
 
+        // https://dev.mysql.com/doc/internals/en/binary-protocol-value.html#packet-ProtocolBinary
+        // MYSQL_TYPE_TINY  = i8
+        // MYSQL_TYPE_SHORT = i16
+        // MYSQL_TYPE_YEAR  = i16
+        // MYSQL_TYPE_LONG  = i32
+        // MYSQL_TYPE_INT24 = i32
         matches!(
             self.column_type(),
-            MYSQL_TYPE_TINY
-                | MYSQL_TYPE_SHORT
-                | MYSQL_TYPE_LONG
-                | MYSQL_TYPE_LONGLONG
-                | MYSQL_TYPE_YEAR
-                | MYSQL_TYPE_INT24
+            MYSQL_TYPE_TINY | MYSQL_TYPE_SHORT | MYSQL_TYPE_YEAR | MYSQL_TYPE_LONG | MYSQL_TYPE_INT24
+        )
+    }
+
+    fn is_int64(&self) -> bool {
+        use ColumnType::*;
+
+        // https://dev.mysql.com/doc/internals/en/binary-protocol-value.html#packet-ProtocolBinary
+        // MYSQL_TYPE_LONGLONG = i64
+        // UNSIGNED MYSQL_TYPE_LONG = u32
+        // UNSIGNED MYSQL_TYPE_INT12 = u32
```

Comment:
Is this correct, reading the implementation underneath says `INT24`.

Reply:
`INT12` doesn't even exist, you're right, I meant `INT24` 👍